resources/views/community/components/developer/dev-games-page.blade.php

```diff
@@ -0,0 +1,185 @@
+@props([
+    'user' => null,
+    'consoles' => [],
+    'games' => [],
+    'sortOrder' => 'console',
+    'soleDeveloper' => false,
+    'userProgress' => null,
+])
+
+<?php
+$makeLink = function($text, $value) use ($sortOrder, $soleDeveloper) {
+    if ($value == '') {
+        // sole developer toggle
+        $suffix = $soleDeveloper ? '' : '&sole=1';
+        return "<a href='?sort=$sortOrder$suffix'>$text</a>";
+    }
+
+    $suffix = $soleDeveloper ? '&sole=1' : '';
+
+    if ($sortOrder === $value) {
+        return "<a href='?sort=-$value$suffix'>$text &#x25B2;</a>";
+    }
+
+    if ($sortOrder == "-$value") {
+        return "<a href='?sort=$value$suffix'>$text &#x25BC;</a>";
+    }
+
+    return "<a href='?sort=$value$suffix'>$text</a>";
+};
```

Comment:
Similar feedback as above, this is a function that accepts some props and returns HTML. Seems like it's actually doing the job of a component.

Reply:
This is just a helper function for managing the sort parameters. I don't think it makes sense as a Blade component. I'm also considering ways to replace the sort implementation with a filter widget similar to what you've done on the beaten list.